test/lint/lint-submodule.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+export LC_ALL=C
+EXIT_CODE=0
+
+CMD=$(git submodule status --recursive)
+if [[ ${CMD} ]]
+then
+    echo These submodules were found, delete them:
+    echo $CMD
```

Comment:
```suggestion
    echo "$CMD"
```

...otherwise, script will print multiple submodules in a line.